### DISCUSSION TITLE
THIS IS AMAZING

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Sid72020123/scratchconnect#6] [General]
This has **so many features!!!** I am gonna try this after my exams are over
 
Have you completed the whole docs?!

How do I get recent comments from a user/project/studio? Without inputting comment ID. Something like:
```python
user.user_comments(Username = "Ankit_Anmol")
user.studio_comments(id="192829283828377371836472892") #studio id
user.projects_comments(id="183829") #something like these exist?
```

And.. u made this module from pypi or github itself?

**Comment by Sid72020123** [comment]
Yes I made that library myself!
To get project comments: 
```python
project.get_comments(all=False, limit=40, offset=0, comment_id=None)
```
Set `comment_id` to `None` if you want comments without using comment ID. Also there are optional parameters like `all`, `limit`, `offset`

I haven't started making the documentation yet because I finished making that library yesterday but I will start making the documentation soon.......

**Reply by Sid72020123** [reply]
There an no features to get user and studio comments but I will start making that soon...

**Reply by Quantum-Codes** [reply]
Thanks :) say me whe it's done! I'm interested
